app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/model/resource-url-manager.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define(['jquery', 'Magento_Checkout/js/model/resource-url-manager'], function(
```

Comment:
Should be converted to mixin that extends `Magento_Checkout/js/model/resource-url-manager`